frontend/src/components/workflows/rawProjections/SubmissionFormRawProjections.tsx

```diff
@@ -0,0 +1,229 @@
+      }
+      if (submittedProjectionIndicesValues.boxesChecked.end) {
+        if (indices !== "") {
+          indices += ", ";
+        }
+        indices += String(lastIndex);
+      }
+    } else if (
+      submittedProjecitonIndicesMethod === ProjectionIndicesMethod.Interval
+    ) {
+      // adds projection frame at start, increments by step and adds one iteratively until stop is reached
+      const start: number =
+        submittedProjectionIndicesValues.intervalValues.start;
+      const stop: number = submittedProjectionIndicesValues.intervalValues.stop;
+      const step: number = submittedProjectionIndicesValues.intervalValues.step;
+      for (let i = start; i < stop; i += step) {
+        indices += i + ", ";
+      }
+      indices += stop;
+    }
+
+    // TODO: error handling here to replace empty strings with null in some cases??
+    const parameters = {
+      ...submittedWorkflowArguments,
+      "projection-indices": indices,
+    };
+
+    function workflowSuccessfullySubmitted(submittedWorkflowName: string) {
+      setWorkflowName(submittedWorkflowName);
+      setWorkflowSubmitted(true);
+    }
+
+    // TODO: Also handle authentication errors here
+    submitWorkflow(visit, parameters, workflowSuccessfullySubmitted);
+  }
+
+  function onWorkflowDataChange(data: WorkflowStatusSubscription$data) {
+    const c = data.workflow.status;
+    if (c === null || c === undefined) {
+      return;
+    }
+    if (
+      c.__typename === "WorkflowErroredStatus" ||
+      c.__typename === "WorkflowFailedStatus"
+    ) {
+      setRetryButtonVisible(true);
+      return;
+    }
+    setRetryButtonVisible(false);
+
+    if ("tasks" in c) {
+      const zipFilesList = c.tasks[0].artifacts.filter(
+        (a) => a.name === "projections.tif"
+      );
+      if (zipFilesList.length !== 0) {
+        setTifURL(
+          c.tasks[0].artifacts.filter((a) => a.name === "projections.tif")[0]
+            .url
+        );
+      }
+    }
+  }
+
+  return (
+    <div>
+      {workflowSubmitted && workflowName !== undefined ? (
+        <div>
+          <WorkflowStatus
+            workflow={workflowName}
+            visit={visitToText(submittedVisit)}
+            onWorkflowDataChange={onWorkflowDataChange}
+          />
+          {retryButtonVisible && (
+            <Button
+              onClick={() => {
+                setWorkflowSubmitted(false);
+                setRetryButtonVisible(false);
+                setSubmittedVisit(undefined);
+                setWorkflowName(undefined);
+              }}
+              variant={"outlined"}
+            >
+              Refill form
+            </Button>
+          )}
+        </div>
+      ) : (
+        <Stack direction="column" spacing={theme.spacing(2)}>
+          <Typography variant="h4" align="center">
+            Workflow:
+            {templateData.title ? templateData.title : templateData.name}
+          </Typography>
```

Comment:
I think this is responsible for the part saying "Workflow:extract-raw-projections" in the page, is that correct? If so, I think we can get rid of this, the users don't need to know the name of the workflow (I understand why this was put in though, it's what the CoR finder page does :slightly_smiling_face:)